### PR TITLE
docs: Update components-scenarios.md

### PR DESCRIPTION
**Pull request by harishkrishnan24**
Fix grammatical issues

## PR Checklist
Please check if your PR fulfills the following requirements:

- [x] The commit message follows our guidelines: https://github.com/angular/angular/blob/main/CONTRIBUTING.md#commit
- [ ] Tests for the changes have been added (for bug fixes / features)
- [x] Docs have been added / updated (for bug fixes / features)

## PR Type
What kind of change does this PR introduce?

- [ ] Bugfix
- [ ] Feature
- [ ] Code style update (formatting, local variables)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] CI related changes
- [x] Documentation content changes
- [ ] angular.io application / infrastructure changes
- [ ] Other... Please describe:

## What is the current behavior?

Issue Number: N/A

## Does this PR introduce a breaking change?

- [ ] Yes
- [x] No

### CODE REVIEW

File: adev/src/content/guide/testing/components-scenarios.md [hunk 1]
```diff
@@ -82,8 +82,8 @@ Here are three tests that illustrate how automatic change detection works.
 The first test shows the benefit of automatic change detection.
 
 The second and third test reveal an important limitation.
-The Angular testing environment does not run change detection synchronously when updates happenthat the test changed the component's `title`.
-The test must call `await fixture.whenStable` to wait for another of change detection.
+The Angular testing environment does not run change detection synchronously when updates happen inside the test case that changed the component's `title`.
+The test must call `await fixture.whenStable` to wait for change detection to triggered.
```
Comment (JeanMeche) [inline adev/src/content/guide/testing/components-scenarios.md:86]: I'd keep that initial sentence.

Comment (harishkrishnan24) [reply adev/src/content/guide/testing/components-scenarios.md:86]: @JeanMeche can you be more clear? You mean you don't want the second line changes?

Comment (JeanMeche) [reply adev/src/content/guide/testing/components-scenarios.md:86]: Yes, it's more about waiting for it to run than for it to be triggered.

**Review by JeanMeche — APPROVED** [review]
Thanks for this !

**Comment by pkozlowski-opensource** [comment]
This PR was merged into the repository by commit 9d941773a265c6a4ed128943a588d0a575e086fb.

The changes were merged into the following branches: main, 18.0.x, 18.1.x

**Pull request closed without merging**